commitlint.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
I have tried another commit message for example `hello world` and it passes

Reply:
what do you mean by `hello world`, did you add some configuration that fixed the issue .?

Reply:
no, what I meant is that it passes anyway even if the commit message doesn't include a word from that array